pkg/generators/forwarding/fluentd/output_fluentd_buffer.go

```diff
@@ -0,0 +1,122 @@
+package fluentd
```

Comment:
Asking because I dont know the right answer... should we introduce an "output" package?

Reply:
It is the right question, we have already 4 go files with `output_` prefix. This is a good indication. I would put this as a follow-up PR/commit. WDYT?

Reply:
It's already under "fowarding" so I would expect that everything in this package is related to "output". Maybe the output_ prefix is unnecessary?

Reply:
Will follow-up on this in a separate PR.